docs/root/operations/admin.rst

```diff
@@ -770,6 +770,12 @@ modify different aspects of the server:
     Text readout stats create a new label value every time the value
     of the text readout stat changes, which could create an unbounded number of time series.
 
+  .. http:get:: /stats?format=prometheus&histogram_emit_mode=histogram,summary
+
+  Optional ``histogram_emit_mode`` query parameter is used to control how histogram metrics get reported.
```

Comment:
I think `histogram mode` is a more concise name.

Reply:
+1

`histogram_mode` I assume though?